src/sentry/user_feedback/blueprints/api.md

```diff
@@ -0,0 +1,280 @@
+| user.username     | optional[string] | -                                                   |
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "browser": {
+          "name": "Chome",
+          "version": "103.0.38"
+        },
+        "contact_email": "colton.allen@sentry.io",
+        "device": {
+          "brand": "Apple",
+          "family": "iPhone",
+          "model": "11",
+          "name": "iPhone 11"
+        },
+        "dist": "abc123",
+        "environment": "production",
+        "error_ids": ["4f0e18332d11431c9b97d924edecb76e"],
+        "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+        "locale": {
+          "lang": "en",
+          "timezone": "UTC+1"
+        },
+        "message": "I really like this user-feedback feature!",
+        "os": {
+          "name": "iOS",
+          "version": "16.2"
+        },
+        "platform": "javascript",
+        "release": "version@1.3",
+        "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+        "sdk": {
+          "name": "sentry.javascript.react",
+          "version": "6.18.1"
+        },
+        "status": "unresolved",
+        "tags": {
+          "hello": "is",
+          "it": ["me", "you're", "looking", "for"]
+        },
+        "timestamp": "2023-08-31T14:10:34.954048",
+        "trace_ids": ["fbb42a39a61e4aa6969b4fecbfbbc114"],
+        "url": "https://docs.sentry.io/platforms/javascript/",
+        "user": {
+          "display_name": "John Doe",
+          "email": "john.doe@example.com",
+          "id": "30246326",
+          "ip": "213.164.1.114",
+          "username": "John Doe"
+        }
+      }
+    ]
+  }
+  ```
+
+### Create User Feedback [POST]
+
+Create a new user-feedback item.
+
+- Request
+
+  ```json
+  {
+    "data": {
+      "dist": "abc123",
+      "contact_email": "colton.allen@sentry.io",
+      "environment": "production",
+      "error_ids": ["4f0e18332d11431c9b97d924edecb76e"],
+      "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+      "message": "I really like this user-feedback feature!",
+      "platform": "javascript",
+      "release": "version@1.3",
+      "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+      "request": {
+        "headers": {
+          "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
+        }
+      },
+      "sdk": {
+        "name": "sentry.javascript.react",
+        "version": "6.18.1"
+      },
+      "status": "unresolved",
+      "tags": {
+        "key": "value"
+      },
+      "timestamp": "2023-08-31T14:10:34.954048",
+      "trace_ids": ["fbb42a39a61e4aa6969b4fecbfbbc114"],
+      "url": "https://docs.sentry.io/platforms/javascript/",
+      "user": {
+        "email": "username@example.com",
+        "id": "123",
+        "ip_address": "127.0.0.1",
+        "name": "user",
+        "username": "user2270129"
+      }
+    }
```

Comment:
```suggestion
  {
    "data": {
      "feedback": {
        "contact_email": "colton.allen@sentry.io",
        "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
        "message": "I really like this user-feedback feature!",
        "url": "https://docs.sentry.io/platforms/javascript/",
        "status": undefined,
        "error_ids": [],
        "trace_ids": [],
      },
      "dist": "abc123",
      "environment": "production",
      "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
      "platform": "javascript",
      "release": "version@1.3",
      "request": {
        "headers": {
          "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
        }
      },
      "sdk": {
        "name": "sentry.javascript.react",
        "version": "6.18.1"
      },
      "tags": {
        "key": "value"
      },
      "timestamp": "2023-08-31T14:10:34.954048",
      "user": {
        "email": "username@example.com",
        "id": "123",
        "ip_address": "127.0.0.1",
        "name": "user",
        "username": "user2270129"
      }
    }
```

Reply:
* we'd be closer aligned with our event schema https://develop.sentry.dev/sdk/event-payloads/types/ -- we separate general user/scope items from our event-type specific data
* imo makes it easier to read/understand - I know that the `feedback` object relates to and comes from our feedback integration